src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
No, you are correct.  Sorry for the confusion.  I agree with the wrappers returning default values for the null type as you suggested above.
```
private static Nullable<FirmwareType> GetFirmwareType()
{
    try
    {
        FirmwareType firmwareType;

        if (Native.GetFirmwareType(out firmwareType))
            return firmwareType;
    }
    catch (Exception)
    {
        // Probably failed to load the DLL or to file the function entry point.
        // Fail silently
    }

    return null;
}
```